### PR TITLE
Bump Nessie to 0.60.1

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: versions.props [hunk 1]
```diff
@@ -48,4 +48,4 @@ com.esotericsoftware:kryo = 4.0.2
 org.eclipse.jetty:* = 9.4.43.v20210629
 org.testcontainers:* = 1.17.6
 io.delta:delta-core_* = 2.2.0
-org.awaitility:awaitility = 4.2.0
\ No newline at end of file
+org.awaitility:awaitility = 4.2.0
```
Comment (Fokko) [inline versions.props:51]: Unrelated change?

**Review by Fokko — APPROVED** [review]

**Review by dimas-b — APPROVED** [review]

File: versions.props (hunk 1 above)
Comment (dimas-b) [reply versions.props:51]: Looks like a line ending change to me.

**Comment by Fokko** [comment]
@snazy yes we are. Thanks @dimas-b for the review
